src/styles/modules/nav-list-iteam-text.scss

```diff
@@ -0,0 +1,19 @@
+.nav-list-item { 
```

Comment:
You have nav-list module and you don't need one more module for nav-list-iteam-text, everything that has nav-list part in it's name should be the part of it module, so please remove this unnecessary module and move this code to your nav-list module

Reply:
done